keps/prod-readiness/sig-network/3726.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 3726
+stable:
```

Comment:
I'm not sure this is necessary at all because we're making 0 code changes and I don't think the KEP actually has a meaningful PRR to review.

Reply:
Thats a very good question! 
I spent time debugging why the tests are failing and apparently it is necessary for every status greater than provisional. 
https://github.com/kubernetes/enhancements/blame/master/pkg/kepval/approval.go#L103